storage/src/testFixtures/java/tech/pegasys/teku/storage/server/rocksdb/InMemoryRocksDbDatabaseFactory.java

```diff
@@ -36,7 +38,22 @@ public static Database createV4(
         new StubMetricsSystem(), hotDb, coldDb, storageMode, stateStorageFrequency);
   }
 
+  public static Database createV6(
```

Comment:
There's a helper class that provides various db versions called `StorageSystemArgumentsProvider` - we should add the v6 dbs to that class.

Reply:
Right! Missed that place